src/apps/workspaces/service/check_access.py

```diff
@@ -85,6 +85,9 @@ async def check_workspace_manager_list_access(self, owner_id: uuid.UUID):
     async def check_applet_manager_list_access(self, applet_id: uuid.UUID):
         await self._check_applet_roles(applet_id, [Role.OWNER, Role.MANAGER])
 
+    async def check_applet_manager_and_limited_list_access(self, applet_id: uuid.UUID):
+        await self._check_applet_roles(applet_id, [Role.OWNER, Role.MANAGER, Role.RESPONDENT])
+
     async def check_workspace_respondent_list_access(self, owner_id: uuid.UUID):
```

Comment:
Take a look in comment above and maybe remove this new method

Reply:
is there. a reason to remove this method I think we need a method with only a respondent access ?
or because we always going to have a respondent in all the cases

Reply:
Please, use already existing one "check_answer_check_access()" as I comment above